_projects/extasy.md

```diff
@@ -4,9 +4,21 @@ collection: projects
 permalink: /projects/extasy
 abstract: "Executing iterative, coupled molecular simulation and analysis kernels on high performance computing systems." 
 status: active
+logo: extasy_logo.png
 ---
 
-The Extensible Toolkit for Advanced Sampling and AnalYsis (ExTASY) is a lightweight software toolkit that supports ensemble-based simulations and their seamless integration with dynamic analysis capabilities and ultra-large time step integration methods. ExTASY enables fast sampling of the conformational dynamics of stable proteins, a prerequisite for the accurate prediction of thermodynamic parameters and biological functions. It also allows tackling systems like intrinsically disordered proteins, which can be beyond the reach of classical structural biology. ExTASY is extensible to other community software components via well-designed and standard interfaces.
-
-<a href="https://bitbucket.org/extasy-project/extasy-workflows"><i class="fa fa-bitbucket"></i>Bitbucket Repository</a><br>
-<a href="https://nsf.gov/awardsearch/showAward?AWD_ID=1265929"><i class="fa fa-nsf"></i>NSF Award</a>
+<div class="project-description">
+	<figure>
+		<img src="../images/projects/extasy.png" width="300">
```

Comment:
Here the user will not need to add the exact name. The page can pick it up from the header
```suggestion
		<img src="{{site.baseurl}}/images/projects/{{page.logo}}" width="250">
```